features/lib/osrm_loader.js

```diff
@@ -78,9 +78,10 @@ class OSRMDirectLoader extends OSRMBaseLoader {
         if (this.osrmIsRunning()) return callback(new Error("osrm-routed already running!"));
 
         this.child = this.scope.runBin('osrm-routed', util.format("%s -p %d", this.inputFile, this.scope.OSRM_PORT), this.scope.environment, (err) => {
-          if (err && err.signal !== 'SIGINT') {
-              throw new Error(util.format('osrm-routed %s: %s', errorReason(err), err.cmd));
-          }
+            if (err && err.signal !== 'SIGINT') {
+                this.child = null;
```

Comment:
Yes, it should be in the master. I changed SIGTERM to SIGINT because for windows npm tests it is possible to check that the OSRM process was killed by `osrmDown` and do not report an error. If test run was successful than `this.child` will be updated in `osrmUp` with the next process. 

This line fixes the problem that if the test failed and not killed than the new process will not be started because `osrmIsRunning()` will return true in `osrmUp`. 

So if one test failed than all next tests will fail also due to 5sec timeout and this is killer to wait appveyor jod for ~ 5000 seconds if almost the first test fails.

If `this.child` set to null here that is ok because the process is already exited then the new process will be started in `osrmUp`.